machine/agema/agema_ag7648c/rootconf/sysroot-bin/onie-diag

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+##
+## 
+## Script to launch platform specific diag.
```

Comment:
I am not entirely clear about what this script is trying to accomplish.  I have not seen other platforms interact with the diag partition in this way.

Can you please explain a bit more what this script is doing?